fdbcli/TenantGroupCommands.actor.cpp

```diff
@@ -76,25 +92,15 @@ ACTOR Future<bool> tenantGroupListCommand(Reference<IDatabase> db, std::vector<S
 			state ClusterType clusterType = wait(TenantAPI::getClusterType(tr));
 			state std::vector<TenantGroupName> tenantGroupNames;
 			state std::vector<std::pair<TenantGroupName, TenantGroupEntry>> tenantGroups;
+			state std::vector<std::pair<TenantGroupName, MetaclusterTenantGroupEntry>> metaclusterTenantGroups;
```

Comment:
This can be written with a direct assignment rather than using state variables and `store`